addons/mail/static/src/core/web/activity_menu.xml

```diff
@@ -20,14 +20,14 @@
                             <div class="flex-grow-1 overflow-hidden">
                                 <div class="d-flex px-2" name="activityTitle" t-out="group.name"/>
                                 <div t-if="group.type === 'activity'" class="d-flex">
-                                    <span t-attf-class="#{group.overdue_count ? '' : 'text-muted'} py-0 px-2 text-truncate" t-on-click.stop="() => this.openActivityGroup(group, 'overdue')">
+                                    <span t-attf-class="#{group.overdue_count ? 'btn-link' : 'text-muted'} py-0 px-2 text-truncate" t-on-click.stop="() => this.openActivityGroup(group, 'overdue')">
```

Comment:
I have a question: Is it okay to use the `btn-link` class without the `btn` class? From what I understand about Bootstrap, `btn-link` is a modifier that sets the visual aspect of the button (background color, border, etc). But, here, it seems like we don't really have a button. So, I wonder if it will be robust to do that. Shouldn't we also add the class `btn` on the element?

:bulb: What about using a standard link (`<a/>`) instead of a span? We can then avoid adding a new class 😃 and the element will naturally look like what we want (just text with an accent color).

Reply:
I'd say as we're doing it in stable it might be best to add the `btn` class instead of completely changing the html element :slightly_smiling_face: